dev/create-release/release-build.sh

```diff
@@ -80,8 +80,17 @@ NEXUS_PROFILE=d63f592e7eac0 # Profile for Spark staging uploads
 BASE_DIR=$(pwd)
 
 MVN="build/mvn --force"
-PUBLISH_PROFILES="-Pmesos -Pyarn -Phive -Phive-thriftserver"
-PUBLISH_PROFILES="$PUBLISH_PROFILES -Pspark-ganglia-lgpl -Pkinesis-asl"
+
```

Comment:
There should be no actual change to the profiles that are set, except that the current Scala 2.11 build adds `-Pkafka-0-8` I rearranged these to make it easier to reason about the profiles in one place.